src/applications/check-in/locales/en/translation.json

```diff
@@ -191,7 +191,14 @@
   "in-person-appointment": "In-person appointment",
   "you-can-still-check-in-with-your-phone-on-the-day-of-your-appointment": "You can still check-in with your phone on the day of your appointment.",
   "telephone-appointment": "Telephone appointment",
+  "we-encourage-you-to-file-travel-reimbursement-now": "We’ll ask you 3 additional questions that can help you file your claim faster. We encourage you to try filing your claim now-even if you’re not sure that you’re eligible.",
   "would-you-like-to-file-a-travel-reimbursement-claim-now": "Would you like to file a travel reimbursement claim now?",
+  "you-are-eligible-for-travel-reimbursement-if-youre": "You are eligible for travel reimbursement if you're traveling for care at a VA health facility or for VA-approved care at a non-VA health facility in your community, and at least one of the following is true:",
+  "you-have-a-va-disability-rating-of": "You have a VA disability rating of 30% or higher.",
+  "youre-traveling-for-treatment-of-a-service-connected-condition": "You're traveling for treatment of a service-connected condition.",
+  "you-receive-va-pension-benefits": "You receive VA pension benefits.",
+  "you-have-an-annual-income-below-the-maximum": "You have an annual income below the maximum annual VA pension rate.",
+  "youre-traveling-in-relation-to-a-compensation-and-pension": "You're traveling to a Compensation and Pension, or C&P, exam.",
```

Comment:
when adding new translations, put them at the end of the file until we get the translations back from the state dept.